test/unit/results_spec.lua

```diff
@@ -140,4 +140,10 @@ describe('Result from running busted', function()
 		expected, results = compute_test_results('single-standalone-error', 'prepend', 'append')
 		assert.are.same(expected, results)
 	end)
+	
+	it('Picks last line with marker', function()
+		local marker = require'neotest-busted._output-handler'.marker
```

Comment:
My style is to have the parentheses when `require` is part of a larger expression.

```suggestion
		local marker = require('neotest-busted._output-handler').marker
```